.grabl/automation.yml

```diff
@@ -32,7 +32,11 @@ build:
     test:
       image: vaticle-ubuntu-20.04
       command: |
-        bazel test //...
+        bazel run //:typedb-extractor -- dist/typedb-all-linux
+        sudo systemd-run ./dist/typedb-all-linux/typedb server
+        sleep 60
+        bazel build //test/integration/...
+        bazel test //test/... --test_output=streamed --spawn_strategy=local
```

Comment:
Now that we have two domains of tests (integration, unit) let's split them up in `automation.yml`:

Introduce `test-unit` and `test-integration`